liquibase-core/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -1031,6 +1031,14 @@ public static byte[] getBytesWithEncoding(String string) {
         return string.getBytes(StandardCharsets.UTF_8);
     }
 
+    /**
+     * @param value string to process
+     * @return string without any whitespaces formatted to lowercase.
+     */
+    public static String toLowerWithoutWhitespaces(String value) {
+        return value.toLowerCase().replaceAll("\\s+", "");
+    }
```

Comment:
Should probably handle the null case here. Maybe something like:

```suggestion
    public static String toLowerWithoutWhitespaces(String value) {
        if (value == null) {
            return null;
        }
        return value.toLowerCase().replaceAll("\\s+", "");
    }
```

Reply:
Nice catch, going to fix!